src/main/webapp/WEB-INF/themes/desktop/sass/pages/_peer-review.scss

```diff
@@ -43,3 +43,14 @@
 .peer-review-accordion-content {
   display: none;
 }
+span.underline {
+  text-decoration: underline;
+}
+
```

Comment:
I recommend removing the redundant `span` selectors--the class names are sufficient on their own.